sce.sh

```diff
@@ -3,7 +3,8 @@
 function print_repo_nicknames {
     echo
     echo "each repo has nicknames:"
-    echo "Core-v4:core-v4, corev4, cv4, c4, c"
+    # echo "Core-v4:core-v4, corev4, cv4, c4, c"
+    echo "Clark:clark, dog, clrk, ck, c"
```

Comment:
remove the commented out line
```suggestion
    echo "Clark:clark, dog, clrk, ck, c"
```